spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/JobExecutionThinController.java

```diff
@@ -86,9 +87,10 @@ public JobExecutionThinController(TaskJobService taskJobService) {
 	 */
 	@RequestMapping(value = "", method = RequestMethod.GET, produces = "application/json")
 	@ResponseStatus(HttpStatus.OK)
-	public PagedModel<JobExecutionThinResource> listJobsOnly(Pageable pageable,
-			PagedResourcesAssembler<TaskJobExecution> assembler) throws NoSuchJobExecutionException {
-		List<TaskJobExecution> jobExecutions = taskJobService.listJobExecutionsWithStepCount(pageable);
+	public PagedModel<JobExecutionThinResource> listJobsOnly(
+			@RequestParam(value = "q", required = false) String queryString,
```

Comment:
Probably want to have something more descriptive than q.

Reply:
Any suggestions? How about "search"? I normally use "q" for search LIKE queries.